Restores the kibana.yml settings for timelion and vis_type_table

stratoula opened the pull request:
## Summary

By removing the Timelion app and the legacy implementation of the datatable viz, we also removed some kibana.yml settings. This means that the clusters that actually have these settings configured will fail. 
We can overpass this by keep registering these settings but declare them as unused. This will log a warning in the server:

![image](https://user-images.githubusercontent.com/17003240/133279890-1afc6c48-b756-4207-bf61-1ed88b04bf00.png)

In v8 we are going to remove them.

kibanamachine commented:
## :green_heart: Build Succeeded
* [continuous-integration/kibana-ci/pull-request](https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/153198/)
* Commit: 0ab2be646e6e7fdb6f1c41de9432dc020953824d
* [Storybooks Preview](https://ci-artifacts.kibana.dev/storybooks/pr-112101/0ab2be646e6e7fdb6f1c41de9432dc020953824d)

### Metrics [[docs](https://www.elastic.co/guide/en/kibana/master/ci-metrics.html)]
> ✅ unchanged

To update your PR or re-run it, just comment with:
`@elasticmachine merge upstream`

<!--PIPELINE
{"builds":[{"status":"SUCCESS","url":"https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/153198/","number":"153198","commit":"0ab2be646e6e7fdb6f1c41de9432dc020953824d"}]}
PIPELINE-->

stratoula commented:
@elasticmachine merge upstream

kibanamachine commented:
merge conflict between base and head

elasticmachine commented:
Pinging @elastic/kibana-vis-editors (Team:VisEditors)

kibanamachine commented:
## :green_heart: Build Succeeded
* [continuous-integration/kibana-ci/pull-request](https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/153780/)
* Commit: bf51d85066dfdc082d5f30f2289e4d864cc90837
* [Storybooks Preview](https://ci-artifacts.kibana.dev/storybooks/pr-112101/bf51d85066dfdc082d5f30f2289e4d864cc90837)

### Metrics [[docs](https://www.elastic.co/guide/en/kibana/master/ci-metrics.html)]
> ✅ unchanged

### History
* :green_heart: [Build #153198](https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/153198/) succeeded 0ab2be646e6e7fdb6f1c41de9432dc020953824d

To update your PR or re-run it, just comment with:
`@elasticmachine merge upstream`

<!--PIPELINE
{"builds":[{"status":"SUCCESS","url":"https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/153198/","number":"153198","commit":"0ab2be646e6e7fdb6f1c41de9432dc020953824d"},{"status":"SUCCESS","url":"https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/153780/","number":"153780","commit":"bf51d85066dfdc082d5f30f2289e4d864cc90837"}]}
PIPELINE-->

flash1293 reviewed (commented):
Seems like this should only go into 7.16, right? I thought we wanted to drop these in 8.0

Probably not part of this PR, but what about removing these renames as well in 8.0?
https://github.com/elastic/kibana/blob/33c124b9d8f09f07ae236098424de08c4ae3827e/src/plugins/vis_types/metric/server/index.ts#L16
https://github.com/elastic/kibana/blob/e20f530762729a8b78a31a1e8b19845bcb83e730/src/plugins/vis_types/table/server/index.ts#L18
https://github.com/elastic/kibana/blob/33c124b9d8f09f07ae236098424de08c4ae3827e/src/plugins/vis_types/tagcloud/server/index.ts#L16
https://github.com/elastic/kibana/blob/33c124b9d8f09f07ae236098424de08c4ae3827e/src/plugins/vis_types/vega/server/index.ts#L20
https://github.com/elastic/kibana/blob/2abbc6f722985e90fb907d42533ab2f04d1a12ec/src/plugins/vis_types/timeseries/server/index.ts#L18

Or do we want to keep them indefinitely?

stratoula commented:
I dont want to tackle the renames in this PR @flash1293 . I will do it in another which will target only v8!
The reason that I added the v8 is because I didn't know what we are going to do with these renames and in the timelion app removal we also removed these:

```
renameFromRoot('timelion_vis.enabled', 'vis_type_timelion.enabled'),
    renameFromRoot('timelion.enabled', 'vis_type_timelion.enabled'),
    renameFromRoot('timelion.graphiteUrls', 'vis_type_timelion.graphiteUrls'),
```
Now that we decided to remove these renames I should create one PR only for 7.x

stratoula commented:
closing in favor of https://github.com/elastic/kibana/pull/112430
